kafka_connect_replicator.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Kafka Connect Replicator Status Finding
+  hosts: kafka_connect_replicator
+  gather_facts: false
+  tags: kafka_connect_replicator
+  environment: "{{ proxy_env }}"
+  tasks:
+    - import_role:
+        name: confluent.variables
+
+    - name: Check if Kafka Service Running
+      shell: "systemctl show -p SubState {{kafka_connect_replicator_service_name}}"
+      changed_when: false
+      check_mode: false
+      register: substate
+
+    - set_fact:
+        install_pattern: "{{ 'parallel' if substate.stdout != 'SubState=running' or kafka_connect_replicator_reconfiguration_pattern == 'parallel' else 'serial' }}"
```

Comment:
change this variable to `kafka_connect_replicator_deployment_strategy` - I recently changed the phrasing